### PR TITLE
Service spec

### LINKED ISSUE #6: API preliminary design

### PR DESCRIPTION
**Pull request by alexhsamuel**
Draft spec for the REST service.  I don't know of a _lightweight_ format for documenting this, so I made one up.  

resolves #6

### CODE REVIEW

**Comment by alexhsamuel** [comment]
@Kelvinm @bereli can you please take a look and comment?  I'd like to work on implementation tonight.

**Comment by Kelvinm** [comment]
Will do, have been pretty swamped today

File: notes/api.md [hunk 1]
```diff
@@ -128,6 +128,10 @@ URL query parameters may be specified to narrow the search:
 - `end=YYYY-MM-DD`: end date
```
Comment (Kelvinm) [inline notes/api.md]: For date formats do we want to provide more than one just in case? YYYYMMDD as well? ( know XX/ZZ/YYYY raises issues)

Comment (alexhsamuel) [reply notes/api.md]: Nooooo!  😢 Let's use the ISO format only.

**Comment by alexhsamuel** [comment]
Thanks @Kelvinm.  For future PRs, shall we get both of the other two to review, and have the author merge?

**Pull request merged**